### PR TITLE
Documentation for AQS

### PR DESCRIPTION
**Pull request by rehrenfeld**

### CODE REVIEW

File: ApplicantQualityScore.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Applicant Quality Score Service is a service helps JobSeekers check how well their profiles match job requirements before applying.
+
+For each job you add, we provide a relevance score:
+
+* 1 - Not Relevant
+* 2 - Low Relevant
+* 3 - Relevant
+* 4 - High Relevant
+
+The Applicant Quality Score Service is available at `/core/optimizer`.
```
Comment (MalarCB) [inline ApplicantQualityScore.md]: oops, i think u forgot to update here

Comment (rehrenfeld) [reply ApplicantQualityScore.md]: thanks :)

**Review by MalarCB — APPROVED** [review]

**Review by JulienStitelet — APPROVED** [review]

File: ApplicantQualityScore.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      }'
+```
+
+Response Structure
+-----------
+Upon a successful execution, the response will have a status code of 200 and a JSON content with the following structure:
+
+job_did (string): The job ID for which the quality score is calculated.
+relevant (number): Indicates the relevance of the applicant to the job.
+
```
Comment (glpayson) [inline ApplicantQualityScore.md:56]: What is the score then? And what is the relationship between score and relevancy? We don't seem to be sorting the results by the score, so what is the score? Is it just another representation of the `relevant` field?

Comment (rehrenfeld) [reply ApplicantQualityScore.md:56]: @glpayson  that's correct, this has been added as an extra field in order to sort them in the future if needed.

**Review by glpayson — COMMENTED** [review]
looks good

**Pull request merged**